sentry-ruby/lib/sentry/attachment.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Sentry
+  class Attachment
+    PathNotFoundError = Class.new(StandardError)
+
+    attr_reader :bytes, :filename, :path, :content_type, :add_to_transactions
+
+    def initialize(bytes: nil, filename: nil, content_type: nil, path: nil, add_to_transactions: false)
+      @bytes = bytes
+      @filename = infer_filename(filename, path)
+      @path = path
+      @content_type = content_type
+      @add_to_transactions = add_to_transactions
```

Comment:
we're still not using `add_to_transactions` in the logic below anywhere.
Now that I think about it, I don't want to have this option in ruby and we can always send the attachment, so I would remove this  and remove the `Event` check in `transport.rb#148`

Reply:
@sl0thentr0py if I remove the Event check, spec breaks because sometimes the event is a plain Hash, what should I do with this?

Reply:
ahh it's Event, sorry yes keep it as is then